src/wallet/wallet.h

```diff
@@ -341,6 +341,11 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
     // ScriptPubKeyMan::GetID. In many cases it will be the hash of an internal structure
     std::map<uint256, std::unique_ptr<ScriptPubKeyMan>> m_spk_managers;
 
+    //! Cache of descriptor ScriptPubKeys used for IsMine. Maps ScriptPubKey to set of spkms
+    std::unordered_map<CScript, std::unordered_set<ScriptPubKeyMan*>, SaltedSipHasher> m_cached_spks;
+    //! Set of which descriptors are not in m_cached_spks
+    std::unordered_set<ScriptPubKeyMan*> m_uncached_spkms;
```

Comment:
I think the names below work best and make the purpose of the variable clearer.

```suggestion
    std::unordered_map<CScript, std::unordered_set<ScriptPubKeyMan*>, SaltedSipHasher> m_ranged_spks;
    //! Set of which descriptors are not in m_cached_spks
    std::unordered_set<ScriptPubKeyMan*> m_non_ranged_spkms;
```

Reply:
I prefer describing the variable with how we expect to use it, rather than what it contains. We're using it as a cache of spkms, the type of spkm doesn't matter.